articles/challenge_participation.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Participating in a Challenge
+layout: article
+excerpt: Learn how to participate in challenges and submit to evaluation queues.
+category: howto
+---
+
+<style>
+#image {
+    width: 100%;
+}
+#toobig {
+    width: 45%;
+}
+</style>
+
+This tutorial will teach you all the steps of participating in a challenge.
+
+## Challenge Registration
+Participants **must** be registered for the challenge if they want to submit and participate.  The registration button can be found on the home page or `How to Participate` page for every challenge.  *In order to be fully registered for any challenge, you must (1) have a Synapse account and become a [certified user](http://docs.synapse.org/articles/getting_started.html#becoming-a-certified-user); (2) agree to the DREAM Rules of participation, and (3) agree to the Terms of Use to work with the Challenge data.*
+
+
+## Join or Create a Team
+Although not mandatory, we encourage you to be part of a team. You can either join a team or create your own team of collaborators. See instructions on how to form your own team of collaborators [here](http://docs.synapse.org/articles/teams.html).  It is important to note that you **cannot** be on more than one team.  Once you have submitted as a team or individual, you cannot submit as another team.  After you form a team, you must register your team to the challenge - there will be a place to do this in every challenge wiki.
+
+## Accessing Challenge Data
+The data stored in the Synapse data repository can be accessed using the Synapse website or programmatically using the Synapse R or Python clients. Instructions on using Synapse are provided in the [Synapse User Guide](http://docs.synapse.org/).  File descriptions are provided on the Data Description page in each challenge wiki.  See instructions 
```

Comment:
Should I provide example pages? Each challenge has different data access restrictions